Adicionado ao maratona-editores o plugin eclipse-cdt, do eclipse.

wallberg13 opened the pull request:
plugins/eclipse-cdt.tar.xz: Arquivo que deve ser adicionado no /etc/skel do
ambiente do maratona, para que na criação do usuário icpc o plugin eclipse-cdt
possa está disponível para o usuário.

d/rules: Copiando o arquivo eclipse-cdt.tar.xz para a pasta /etc/skel

d/maratona-editores.postinst: Adicionado ao script a parte referente ao
eclipse-cdt, que é de descompactar o arquivo no /etc/skel.

Signed-off-by: Wall Berg Morais <wallbergmirandamorais@gmail.com>

wallberg13 committed:
Adicionado ao maratona-editores o plugin eclipse-cdt, do eclipse.

plugins/eclipse-cdt.tar.xz: Arquivo que deve ser adicionado no /etc/skel do
ambiente do maratona, para que na criação do usuário icpc o plugin eclipse-cdt
possa está disponível para o usuário.

d/rules: Copiando o arquivo eclipse-cdt.tar.xz para a pasta /etc/skel

d/maratona-editores.postinst: Adicionado ao script a parte referente ao
eclipse-cdt, que é de descompactar o arquivo no /etc/skel.

Signed-off-by: Wall Berg Morais <wallbergmirandamorais@gmail.com>

bcribas commented on debian/maratona-editores.postinst:
```diff
@@ -48,3 +48,7 @@ while [ $flag -eq 1 ]; do
+# Adicionando o plugin do Eclipse - Descompactando o arquivo.
+[ -f /etc/skel/eclipse-cdt.tar.xz ] && \
+tar -xf /etc/skel/eclipse-cdt.tar.xz -C /etc/skel/ && \
+rm /etc/skel/eclipse-cdt.tar.xz
```
isso vai deixar o pacote em um estado inconsistente. Não se pode apagar arquivos de um pacote se o pacote não será removido.

O melhor é já entrar com os plugins descompactado no pacote.

bcribas commented on debian/rules:
```diff
@@ -1,6 +1,10 @@
 override_dh_auto_install:
+	# Plugin Eclipse-cdt
+	mkdir -p debian/maratona-editores/etc/skel/
+	cp plugins/* debian/maratona-editores/etc/skel/
```
Seguindo a linha do comentário anterior, aqui já descompacta o que tem em plugins no diretório skel

wallberg13 committed:
Correções na instalação do plugin eclipse-cdt.

d/rules: No próprio rules agora é feita a descompactação do arquivo
eclipse-cdt.tar.xz.

d/maratona-editores.postinst: Removido trecho do script que descompacta o
arquivo eclipse-cdt.tar.xz e em seguida remove o arquivo, deixando assim o
pacote em um estado consistente.

Signed-off-by: Wall Berg Morais <wallbergmirandamorais@gmail.com>
